prow/cmd/webhook-server/helpers.go

```diff
@@ -90,7 +98,7 @@ func genCert(expiry int, dnsNames []string) (string, string, string, error) {
 		DNSNames:     dnsNames,
 		SerialNumber: big.NewInt(1658), //unique identifier for cert
 		Subject: pkix.Name{
-			CommonName:   "validation-webhook-service.default.svc",
+			CommonName:   "admission-webhook-service.default.svc",
```

Comment:
This service name doesn't match with anything from this service or webhook configs. As discussed offline, please add a comment here stating that this name doesn't really matter